ui/Logo.tsx

```diff
@@ -0,0 +1,42 @@
+import { cva } from "class-variance-authority";
+import Image from "next/image";
+import { twMerge } from "tailwind-merge";
+
+export const logoStyles = cva(
+  ["flex items-center justify-center", "rounded-100 bg-surface-white"],
+  {
+    variants: {
+      size: {
+        lg: ["w-10 h-10"],
+        md: ["w-8 h-8"],
+        sm: ["w-6 h-6"],
+        xs: ["w-5 h-5"],
+        "2xs": ["w-4 h-4"],
+      },
```

Comment:
maybe we could use the new size prop

https://tailwindcss.com/docs/size

Reply:
Will use it, thanks!